remark/mermaid/style.css

```diff
@@ -9,18 +9,54 @@
   /* https://github.com/seek-oss/braid-design-system/blob/braid-design-system%4032.23.1/packages/braid-design-system/src/lib/color/palette.ts */
   --braid-blue-100: #e3f2fb;
   --braid-blue-300: #98c9f1;
+  --braid-blue-500: #0d3880;
   --braid-blue-700: #1d559d;
   --braid-grey-50: #f7f8fb;
   --braid-grey-100: #eaecf1;
+  --braid-grey-200: #d2d7df;
   --braid-grey-300: #abb3c1;
   --braid-grey-500: #5a6881;
+  --braid-grey-700: #2e3849;
+  --braid-grey-800: #1c2330;
   --braid-mint-100: #e2f7f1;
   --braid-mint-300: #8bdec5;
+  --braid-mint-500: #28b888;
   --braid-mint-700: #12784f;
+  --braid-red-100: #ffe3e2;
   --braid-red-300: #fb999a;
+  --braid-red-500: #f94344;
   --braid-red-700: #b91e1e;
   --braid-yellow-100: #fef8de;
+  --braid-yellow-300: #fee384;
   --braid-yellow-500: #fdc221;
+  --braid-yellow-700: #b9800d;
+
+  /* https://github.com/seek-oss/braid-design-system/blob/braid-design-system%4032.23.1/packages/braid-design-system/src/lib/themes/baseTokens/apac.ts#L51 */
```

Comment:
Lazy comment but should we be referencing seekJobs values?

Reply:
It's a good question. The `seekJobs` theme is largely based on the `apac` base tokens but makes a few customisations, so it makes sense to link both (and I missed one `75`):

```suggestion
  /* https://github.com/seek-oss/braid-design-system/blob/braid-design-system%4032.23.1/packages/braid-design-system/src/lib/themes/baseTokens/apac.ts#L51 */
  /* https://github.com/seek-oss/braid-design-system/blob/braid-design-system%4032.23.1/packages/braid-design-system/src/lib/themes/seekJobs/tokens.ts#L140-L189 */
```

There are some creative liberties taken though—we don't exactly map 1:1.